doc/developer-documentation/migration-to-new-version.md

```diff
@@ -0,0 +1,127 @@
+# Migration guide from Etherlime v1.2 to v2.0
+
+## Breaking Changes
+
+### Installing v1.2
+
+```text
+npm i -g etherlime
+```
+This command in v1.2 fetch and install the whole library.
+
+### Installing v2.0
+
+#### Installing the CLI: 
+```text
+npm i -g etherlime
+```
+
+This command in v2.0 will install **only** the etherlime CLI commands library. You can use all the [CLI](https://app.gitbook.com/@etherlime/s/etherlime/developer-documentation/etherlime-cli) commands.
+
+
+#### Installing the LIB: 
+```text
+npm i -g etherlime-lib
```

Comment:
Maybe it's better not to use the global flag. That way the project that uses etherlime-lib will have it as a dependency in the package.json.